src/main/resources/assets/iris/lang/ru_ru.json

```
@@ -6,9 +6,12 @@
   "iris.shaders.debug.disabled": "Отладка отключена.",
   "iris.shaders.debug.failure": "Невозможно включить отладку, ваш компьютер не поддерживает необходимые функции.",
   "iris.shaders.debug.restart": "Отсутствует отладочный контекст. Перезапустите игру, чтобы включить отладку.",
+  "iris.load.failure.shader": "Не удалось загрузить набор шейдеров! Пожалуйста, сообщите об ошибке разработчику шейдера.",
```

```suggestion
  "iris.load.failure.shader": "Не удалось загрузить набор шейдеров, сообщите об ошибке его разработчику.",
```
Обычно в локализациях на русский принято опускать английский формализм. Также рекомендую использовать подобную конструкцию, чтобы избежать тавтологии.